internal/cmd/kafka/command_cluster_describe.go

```diff
@@ -36,7 +37,7 @@ type describeStruct struct {
 	ByokKeyId          string `human:"BYOK Key ID" serialized:"byok_key_id"`
 	EncryptionKeyId    string `human:"Encryption Key ID" serialized:"encryption_key_id"`
 	RestEndpoint       string `human:"REST Endpoint" serialized:"rest_endpoint"`
-	TopicCount         int    `human:"Topic Count" serialized:"topic_count"`
+	TopicCount         int    `human:"Topic Count,omitempty" serialized:"topic_count,omitempty"`
```

Comment:
This would technically be a breaking change, so I'd rather not apply it to the serialized output. Can you make a V4 ticket for this?
```suggestion
	TopicCount         int    `human:"Topic Count,omitempty" serialized:"topic_count"`
```

Reply:
Will do. In this case, the serialized output should just report 0, right?

Or perhaps -1?